### ISSUE TITLE
Add support for Eskom Schedule (similar to COCT feature)

### ISSUE THREAD

**Issue by tiaanv** [tiaanv/ha-eskom-loadshedding#1]
Add Full Eskom API to implement the schedule for Eskom customers in the same manner that the COCT schedule is supported.
Will use the Eskom API for this
Toying with idea that you just enter the suburbID.  Then have a seperate method to find the suburb ID.
Alternative is implementing the whold province/suburb search, but that's quite a bit more work.. perhaps scope in 2nd phase.

**Comment by cschutte** [comment]
Yes - I agree - the whole process of finding a suburbID is only done once, and you can provide instructions for that. It will just add unnecessary complexity now to the code if you add the search function - but will sure make sense in the long run. Will be great if you can get Eskom also going! 

BTW - the new update you loaded this morning for COCT is working great! A pity Eskom stopped Load Shedding last night - now we will have to wait to test it properly! but guess we will not have to wait long... ;-)

**Comment by DravenSA** [comment]
Shame this never happened, Would really like it for CoJ

**Comment by tiaanv** [comment]
> Shame this never happened, Would really like it for CoJ

@DravenSA  there is another clever body who has done it for you ;) Go check it out.
[https://github.com/wernerhp/ha_integration_load_shedding](https://github.com/wernerhp/ha_integration_load_shedding)

We are going to combine the two soon, but in the interim, there is a working solution for you.

**Comment by DravenSA** [comment]
> > Shame this never happened, Would really like it for CoJ
> 
> @DravenSA there is another clever body who has done it for you ;) Go check it out. https://github.com/wernerhp/ha_integration_load_shedding
> 
> We are going to combine the two soon, but in the interim, there is a working solution for you.

Unfortunately this will not add as an integration, I have sent a request, but there is nothing, 
https://github.com/wernerhp/ha.integration.load_shedding/issues/15

but thanks for the reply
